USAGE.md

```diff
@@ -249,6 +249,32 @@ You can prefix local files with a dot to define them relative to `packwerk.yml`,
 You can also reference the name of a gem.
 
 ## Examples
+### Custom `OffensesFormatter`
+While `packwerk` ships with its own `OffensesFormatter`, the extension system provides a mechanism to configure your own to be used when `bin/packwerk check` is run.
+
+Firstly, you'll need to create an `OffensesFormatter` that implements the `Packwerk::OffensesFormatter` interface. You can use [`Packwerk::Formatters::OffensesFormatter`](lib/packwerk/formatters/offenses_formatter.rb) as a model for this. Then, in the `require` directive described above, you'll want to tell `packwerk` about it:
+```ruby
+# ./path/to/file.rb
+class MyOffensesFormatter
+  include Packwerk::OffensesFormatter
+  # implement the `OffensesFormatter` interface
+
+  def identifier
+    'my_offenses_formatter'
+  end
+end
+```
+
+Then in `packwerk.yml`, you can set the `formatter`:
+```yml
+formatter: my_offenses_formatter
```

Comment:
Should this be `offenses_formatter`? Or just `format` to match the CLI flag?

Reply:
Either the config nor CLI flag currently exist to my knowledge so shouldn't be breaking, so down to call it whatever!

Reply:
Ah, wrote this with renaming the constant in mind, but if we go with the identifier approach you are correct 👍 

Let's go with `offenses_formatter` like you did with the CLI arg and we can change it before release if need be.